roles/ansible_service_broker/vars/openshift-enterprise.yml

```diff
@@ -1,9 +1,9 @@
 ---
 __ansible_service_broker_registry_type: rhcc
 __ansible_service_broker_registry_name: rh
-__ansible_service_broker_registry_url: "https://registry.access.redhat.com"
-__ansible_service_broker_registry_user: null
-__ansible_service_broker_registry_password: null
+__ansible_service_broker_registry_url: "{{ oreg_url | default('https://registry.access.redhat.com') }}"
```

Comment:
it would be good if we could get this default (registry.access.redhat.com) from a single global location(particularly since we're about to change it), not sure if we have one today though.

Reply:
Agreed. I think it's best to track that improvement separately from this one.